api/v1alpha1/gitlabidentityprovider_types.go

```diff
@@ -117,7 +119,7 @@ type GitLabIdentityProvider struct {
 
 //+kubebuilder:object:root=true
 
-// GitLabIdentityProviderList contains a list of GitLabIdentityProvider
+// GitLabIdentityProviderList contains a list of GitLabIdentityProvider.
```

Comment:
Not a blocker, but why is a comment like this useful at all?

Reply:
This is scaffolded from Kubebuilder.  Easier to leave them in place than remove them.